docs/user-guide/component/packer.md

```diff
@@ -53,9 +54,10 @@ await resource.run(ZipPacker)
 
 ### Run Automatically
 Packers can also be run automatically against resources with valid resource tags. For example, consider the following code:
+
 ```python
 from ofrak.resource import Resource
-from ofrak_components.zip import ZipArchive
+from ofrak.core.zip import ZipArchive
```

Comment:
These imports also almost certainly have to be fixed in the GUI. Lines like the following will need to be changed:

https://github.com/redballoonsecurity/ofrak/blob/e1386b54a08a133d63f03726248cfc76f3f19744/frontend/src/AttributesView.svelte#L26